ssl-config/src/main/java/com/palantir/remoting/ssl/KeyStoreConfiguration.java

```diff
@@ -33,7 +32,7 @@
 public abstract class KeyStoreConfiguration {
 
     @Value.Parameter
-    public abstract Path path();
+    public abstract URI path();
```

Comment:
I'm not sure why these are jackson serializable anyway. Path and URI aren't
exactly to/from jsonifiable